iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java

```diff
@@ -971,7 +945,7 @@ public static void createDevice(
             .execute();
 
     System.out.println("Created device: " + createdDevice.toPrettyString());
-    // [END create_device]
+    // [END iot_create_device]
```

Comment:
You missed the:

* create_gateway
* list_gateways
* list_devices_for_gateway

functions below

Reply:
Ahh, i will add that real quick